### PR TITLE
Update tf_resizeetc.sp

### PR DESCRIPTION
**Pull request by Drixevel-Archive**
Added 2 default ConVars so you can set the default size on connect/command usage. Also added 2 commands that can be modified to set your own hand/torso sizes.

### CODE REVIEW

**Comment by nosoop** [comment]
Hi!

Meaning to add commentary on usage; from here it looks like sm_resizemyhand only sets it to the default size defined by `sm_resize_torso_default` and `sm_resize_hand_default`?  Was hoping for something more like `sm_resizemyhand <scale, default scale if empty>`.

Besides that, hConVars[] seems fairly generic and unreadable if more convars are added -- I wouldn't want to have to dig back into the comments to figure out what `hConVars[4]` stands for.  I'd suggest declaring `Handle:g_hCvarDefaultHandScale = INVALID_HANDLE;` ("global referring to a cvar handle for default hand scale") and `Handle:g_hCvarTorsoScale = INVALID_HANDLE` closer to the lines declaring the arrays for all clients (line 20).  Personal preference; most everything besides includes should be placed below the myinfo section.

Also with that, the "default" values for said cvars should be "1.0".  Don't want people installing it and having to reconfigure to unscaled hands / torso.

Basically, good ideas being implemented, but not up to par for my standards of code readability and usability in the plugin as-is; sorry!  To summarize, if you're willing to fix the following...
- Cleanly name the variables for the convar handles
- Set said cvar values to 1.0 as a sane default
- Move the cvar declarations below the `myinfo` struct
- Fully implement the `sm_resizemy*` commands

I'd be happy to pull the changes in then.  I don't want to come off as an ass; just want to control the code quality.

Thanks for making the request as well!  Working on GitHub is a learning experience for me; I'll be sure to write up a guide for my code styling for future reference.

**Comment by Drixevel-Archive** [comment]
It's meant to be an easy method of building off the code. I was asked by someone else to add those features so I did.

The reasons I added the 'hConVars[]' handle system is because I tend to work on plugins with a large amounts of lines and a large amounts of ConVars, It's more of a case of the normal.
